docs/4.4/api-reference.md

```diff
@@ -796,14 +798,111 @@ if err := client.DeleteAccessRequest(ctx, accessReqID); err != nil {
+}
+if err := client.RotateCertAuthority(req); err != nil {
+  return err
+}
+```
+
+**Rotation Phases**
+
+In `auto` mode, the rotation is scheduled to run in a series of phases. These phases occur in the order below, unless you set a custom schedule, or run each phase manually.
+
+1. `Standby`: No rotation operations underway.  This is the beginning and end state of every CA rotation. If a CA's rotation is not in the standby state, a new rotation cannot begin.
+2. `Init`: New Certificate Authority is issued, but it remains unused while users and servers get updated certificates.
+3. `Update Clients`: Client credentials will have to be updated and reloaded, but servers will still use and respond with old credentials (grace period).
+4. `Update Servers`: Servers will have to reload and should start serving TLS and SSH certificates signed by new CA (retrieved in previous phase).
+5. `Rollback`: Rolls the CA back to the old CA. This phase is not triggered automatically, but can be used in the case of a rotation failure. e.g. if your servers go down during the rotation, and aren't up in time within the grace period, leaving them with only the old certificates.
+
+**Rotation Schedule**
+
+You can also rotate certificates with a custom schedule. The default schedule gives the `Init`, `UpdateClients`, and `UpdateServers` phases each 1/3 of the total grace period to complete. You may want to extend the length of one phase in the schedule without having a ridiculously long grace period, since long grace periods present a possible vulnerability
+
+```go
+// This will automate your CA rotation with the custom schedule. Use with caution, each phase should have times extra time to ensure they complete in less than optimal situations.
+if err := client.RotateCertAuthority(auth.RotateRequest{
+  Mode: services.RotationModeAuto,
+  Schedule: &services.RotationSchedule{
+    // 1 hour for Init
+    UpdateClients: time.Now().UTC().Add(time.Hour).UTC(),
+    // 4 hours for UpdateClients
+    UpdateServers: time.Now().UTC().Add(time.Hour * 5).UTC(),
+    // 2 hours for UpdateServers
+    Standby: time.Now().UTC().Add(time.Hour * 7).UTC(),
+  },
+}); err := client.RotateCertAuthority(req); err != nil {
+  return err
+}
+```
+
+**Manual Rotation** (custom automated solution)
+
+The best option is to set up a custom system for auth rotation using `manual` mode, and triggering each phase manually, so that the rotation does not depend on an arbitrary rotation schedule based on an arbitrarily long grace period.
+
+You can set up a custom automated solution using `manual` mode where each phase is triggered by the event of it completing. For example, if you are tracking a cluster's servers and know when they have all updated to the new certificates, you can trigger the next phase. This is both quicker, and more error proof in cases where the servers go offline and don't have time to update their certificates.
```

Comment:
are tracking ~~a~~ cluster's servers

By triggering the next phase do you mean manual use of specific APIs from a script? Also, it's a bit unclear about tracking event completions - would it be possible to expand on this?

Reply:
I added a note that I meant to use the RPC listed below to trigger the next phase. As for the tracking event completion, that was more of a guess of what a good use case for manual rotations would be useful for. I'll take another stab at making this more straightforward on less based on conjecture.